docs/GettingStartedDocs/install_oe_sdk-Ubuntu_16.04.md

```diff
@@ -38,7 +38,7 @@ If the output of the above is blank, you should proceed with installing the driv
 ```bash
 sudo apt update
 sudo apt -y install dkms
-wget https://download.01.org/intel-sgx/sgx-dcap/1.7/linux/distro/ubuntu16.04-server/sgx_linux_x64_driver_1.35.bin -O sgx_linux_x64_driver.bin
+wget https://download.01.org/intel-sgx/sgx-dcap/1.8/linux/distro/ubuntu16.04-server/sgx_linux_x64_driver_1.35.bin -O sgx_linux_x64_driver.bin
```

Comment:
The 1.35 driver is under 1.7. Going to this leads to a 404.

https://github.com/openenclave/openenclave/search?l=YAML&q=1.3

Reply:
Updated!